app.js

```diff
@@ -1,46 +1,93 @@
 'use strict';
+//
+var score = 0;
 
-var questionOne = 'Have I ever lived in a foriegn country?';
+var greeting = ('Hello, user. I would like you to play a guessing game about me. To begin, please tell me your name.');
+var userName = prompt(greeting);
+
+var questionOne = 'Okay, ' + userName + ' Have I ever lived in a foriegn country?';
 var country = prompt(questionOne).toLowerCase();
 if (country === 'yes' || country === 'y') {
   alert ('You are Correct! I lived in the Middle East country of Bahrain.');
```

Comment:
You don't need a space between the name of the function that you're calling and the parentheses you are using to call it.